frontend/views/containers/sidebar/GroupMembers.vue

```diff
@@ -49,16 +81,23 @@ export default {
     MenuTrigger,
     MenuContent,
     MenuItem,
+    Tooltip,
     UserImage
   },
   methods: {
     invite () {
       this.$router.push({ path: '/invite' })
+    },
+    openModal (modal) {
+      sbp('okTurtles.events/emit', LOAD_MODAL, modal)
     }
   },
   computed: {
     profiles () {
       return this.$store.getters.profilesForGroup()
+    },
+    profilesCount () {
+      return Object.keys(this.profiles).length
```

Comment:
Use `...mapGetters['memberCount']` instead (DRY)

Reply:
Oh, cool, done. In addition, I did the same for the getter `profilesForGroup` and realized that it was a getter not well written.

(@pieer please correct me here)

Getters in Vuex work as `computed` properties in Vue. Computed properties should update its value automatically as "derived values" and should not return callbacks (another function) but `profilesForGroup` was doing that. That explains why doing `...mapGetters['profilesForGroup']` didn't work. So I fixed it.

Also, to improve consistency in naming I renamed some getters related to groups:
- profilesForGroup -> groupMembers
- memberUsernames -> groupMembersByUsername
- memberCount -> groupMembersCount

Reply:
I'm not sure to understand: why you can't use ...mapGetters['memberCount']
Or maybe the file is not updated?

Reply:
I assume that meanwhile, you realized that you were seeing the wrong files x)

Reply:
Yes, it's all good now